src/main/scala/cromwell/engine/backend/jes/Run.scala

```diff
@@ -37,12 +39,29 @@ object Run  {
     def toRunMap = params.map(p => p.name -> p.gcs).toMap.asJava
   }
 
+  private def dateTryToString(dateTry: Try[DateTime]): String = dateTry match {
+    case scala.util.Success(d: DateTime) => d.toString
+    case Failure(e: Throwable) => e.getClass.getName
+  }
+
   sealed trait RunStatus
   trait TerminalRunStatus extends RunStatus
-  final case class Initializing(created: DateTime) extends RunStatus
-  final case class Running(created: DateTime, started: DateTime) extends RunStatus
-  final case class Success(created: DateTime, started: DateTime, finished: DateTime) extends TerminalRunStatus
-  final case class Failed(created: DateTime, started: DateTime, finished: DateTime, errorCode: Int, errorMessage: String) extends TerminalRunStatus
+  final case class Initializing(created: Try[DateTime]) extends RunStatus {
+    override def toString = s"Initializing(created: ${dateTryToString(created)})"
+  }
+
+  final case class Running(created: Try[DateTime], started: Try[DateTime]) extends RunStatus {
+    override def toString = s"Running(created: ${dateTryToString(created)}, started: ${dateTryToString(started)})"
+  }
+
+  final case class Success(created: Try[DateTime], started: Try[DateTime], finished: Try[DateTime]) extends TerminalRunStatus {
+    override def toString = s"Success(created: ${dateTryToString(created)}, started: ${dateTryToString(started)}, finished: ${dateTryToString(finished)})"
+  }
+
+  final case class Failed(created: Try[DateTime], started: Try[DateTime], finished: Try[DateTime], errorCode: Int, errorMessage: String) extends TerminalRunStatus {
+    override def toString = s"Failed(created: ${dateTryToString(created)}, started: ${dateTryToString(started)}, finished: ${dateTryToString(finished)})"
+  }
+
```

Comment:
It's because this is intentionally an ADT and the product types contained by the sum type each have those values. That's not duplicating data (well, it is ...), it's a proper sum/product type combination.

That said, see my comments elsewhere that this whole enchilada should be changed radically